README.md

```diff
@@ -369,7 +369,7 @@ Hurray! You successfully made a contribution!
   - [Praveen Kumar Purushothaman](https://github.com/praveenscience/)
   - [Prithviraj Biswas](https://github.com/prithvirajbytes)
   - [Puneeth P Gowda](https://github.com/puneeth-techie)
-
+  - [Pranav Pratap Singh](https://github.com/Pranav108)
```

Comment:
Please rearrange your name according to alphabetical order. Your name should be below Pranav Kotak (Line no. 364)